lib/Map/TableStructure.js

```diff
@@ -46,7 +46,7 @@ var defaultDisplayVariableTypes = [VarType.ENUM, VarType.SCALAR, VarType.ALT];
  */
 var TableStructure = function(name, options) {
     options = defaultValue(options, defaultValue.EMPTY_OBJECT);
-    DisplayVariablesConcept.call(this, name, options.getColorCallback);
+    DisplayVariablesConcept.call(this, name, options.getColorCallback, options.disableColumnsWithDifferentXType);
```

Comment:
Actually it disables columns from other tables, which is a bit lost with this name.
To also make it consistent with `getColorCallback`, how about `disableIncompatibleTableColumnsCallback`?

I don't think this ever worked actually, as the constructor's signature is:
```var DisplayVariablesConcept = function(name, allowMultiple, getColorCallback)```

Also - I don't see where this is ever stored by `DisplayVariablesConcept`?